healtcheck.go

```diff
@@ -0,0 +1,52 @@
+package server
+
+import (
+	"github.com/juju/errgo"
+)
+
+const (
+	StatusHealthy   = "healthy"
+	StatusUnhealthy = "unhealthy"
+)
+
+type HealthInfo struct {
+	Status   string       `json:"status"`
+	App      string       `json:"app"`
+	Version  string       `json:"version"`
```

Comment:
do we want to have `omitempty` for this? Afaik the only required field here is `status`.